platform/ios/BUILD.bazel

```diff
@@ -158,6 +158,7 @@ objc_library(
     name = "sdk",
     hdrs = _PUBLIC_HEADERS,
     data = ["resources"],
+    module_name = "Mapbox",
```

Comment:
Do we want to rename this?

Reply:
I can rename this.

It affects how you import it in Swift code.